### ISSUE TITLE
arm64 version for mac silicon M1

### ISSUE THREAD

**Issue by gadelkareem** [confluentinc/kafka-connect-datagen#109]
Docker has a memory leak problem with non amr64 containers https://github.com/docker/for-mac/issues/5204#issuecomment-876289068

Can we get an arm64 build?

**Comment by conradwt** [comment]
Yes, I'm also looking for an arm64 build of this image.  BTW, I also experienced a memory link yesterday where I received an out-of-memory warning from macOS 12.5.1.
